frontends/php/include/classes/items/CHelpItems.php

```diff
@@ -142,7 +142,7 @@ protected function getItems() {
 				],
 				[
 					'key' => 'proc.cpu.util[<name>,<user>,<type>,<cmdline>,<mode>,<zone>]',
-					'description' => _('Process CPU utilisation percentage. Returns float')
+					'description' => _('Process CPU utilization percentage. Returns float')
```

Comment:
This is an example of an en-GB -> en-US change, but note that the en-US form was already dominant.